### DISCUSSION TITLE
account

### DISCUSSION THREAD

**Discussion by vishva98432** [vercel/community#3854] [Account Issue] [answered]
### Question

why my  account is blocked

**Comment by amyegan** [comment, marked as answer]
Hey @vishva98432. Sorry you're having trouble signing up for a new account.  We can't safely discuss account information in public because of security concerns..

You may have received an email with more details about the error and instructions on next steps. For more info, please check this guide: [Why am I unable to login or signup to the Vercel platform?](https://vercel.com/guides/why-can-i-not-signup)
